### DISCUSSION TITLE
Fine-tuning on a single time series

### DISCUSSION THREAD

**Discussion by teshnizi** [amazon-science/chronos-forecasting#118] [Q&A]
Hello,

I only have a single time series, and I want to do forecasting on it. Does it make sense to do fine-tuning in this case?

I was thinking maybe I could split the data chronologically (use data from 2020 to 2022 for training and data from 2022 to 2024 for testing), but I'm not sure if that makes sense.

Thanks

**Comment by lostella** [comment]
@teshnizi I think fine-tuning on a single time series may not be ideal. First it's worth doing some backtesting on the series using the pretrained model, and see how accurate it appears to be on historical data. Did you try that already?

**Comment by teshnizi** [comment]
Thanks for your reply @lostella. I see.
I did try it out-of-the-box, but the results I got after fine-tuning it on the first part of the time series were better (It went from like 20% to 10% error rate on the unseen data). It overfits when I tune it for more than a couple thousand iterations. 

Our of curiosity, Have you guys thought about potential extensions to the model that would allow for the use of domain knowledge?

**Reply by kanghui-learning** [reply]
Hi, I fine-tuned some classical time series datasets, such as ETT, but did not get positive results(It always overfit the data). I’m curious about the dataset you worked on and the size of the model you used. Could you provide more details?

**Comment by lostella** [comment]
@teshnizi not concretely, I believe, but I think it's an interesting direction!

**Comment by teshnizi** [comment]
I'd love to have a chat and explore potential future directions if that is of interest to you.
